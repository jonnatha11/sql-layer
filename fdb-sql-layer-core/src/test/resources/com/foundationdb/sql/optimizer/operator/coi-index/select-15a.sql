SELECT * FROM orders INNER JOIN items ON orders.oid = items.oid
 WHERE (orders.order_date = '2000-10-31' OR orders.order_date = '2000-11-01')
   AND (items.iid = 100 OR items.iid = 200)