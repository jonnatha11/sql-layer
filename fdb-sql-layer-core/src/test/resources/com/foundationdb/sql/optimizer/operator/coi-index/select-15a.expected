PhysicalSelect[oid:int, cid:int, order_date:date, iid:int, oid:int, sku:varchar(32), quan:int]
  Project_Default(orders.oid, orders.cid, orders.order_date, items.iid, items.oid, items.sku, items.quan)
    Flatten_HKeyOrdered(orders INNER items)
      GroupLookup_Default(Index(items.PRIMARY) -> orders, items)
        Intersect_Ordered(skip 1 left, skip 1 right, compare 2)
          Union_Ordered(all, skip 1, compare 2)
            IndexScan_Default(Index(items.PRIMARY), iid = 100, orders.cid ASC, oid)
            IndexScan_Default(Index(items.PRIMARY), iid = 200, orders.cid ASC, oid)
          Union_Ordered(skip 1, compare 2)
            IndexScan_Default(Index(orders.order_date), order_date = DATE '2000-10-31', cid ASC, oid)
            IndexScan_Default(Index(orders.order_date), order_date = DATE '2000-11-01', cid ASC, oid)